.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [ ] Kindly, remove this console.log debugging statements present in these files:

![image](https://github.com/Swarzstein/final-capstone/assets/57421823/07320298-7474-4778-9acd-79aca86fc11f)

That way, the repo will be cleaner. 👍